packages/client/src/internals/httpRequest.ts

```diff
@@ -15,17 +14,17 @@ function arrayToDict(array: unknown[]) {
 export function httpRequest<TResponseShape = TRPCResponse>(
   props: {
     runtime: LinkRuntimeOptions;
-    type: ProcedureType;
+    type: 'query' | 'mutation';
+    method: 'query' | 'mutation';
```

Comment:
I think `'get' | 'post'` Are more intuitive here don’t you think?

Reply:
Option A
```suggestion
    method: 'GET' | 'POST';
```
Option B
```suggestion
    typeOverride: 'query' | 'mutation';
```

Thoughts?

Reply:
I prefer A, it more directly tells the end user what the property will actually do. Aside from changing get to post it doesn’t somehow turn a query into a mutation. It’s purely an http thing right?